redbeat/schedulers.py

```diff
@@ -163,13 +165,20 @@ def __init__(self, app=None):
         self.schedule_key = self.key_prefix + ':schedule'
         self.statics_key = self.key_prefix + ':statics'
         self.lock_key = self.either_or('redbeat_lock_key', self.key_prefix + ':lock')
+        self.lock_token = self.generate_lock_token()
```

Comment:
I think this token should be generated at acquire time to keep the same semantics we currently have. Also I would guess including the token in the info log below the acquire line would be good since ideally you would want to be able to check your logs to figure out which host has the lock rather than accessing redis for that information.

Ideally we'd also want to move towards using the stock redis-py Lock implementation since it now has a reacquire method that has the behaviour we're overriding extend with.